src/component/AllComponnts.jsx

```diff
@@ -0,0 +1,3 @@
+export {Footer} from "./Footer"
```

Comment:
suggestion: `AllComponents` file name is misspelled

Reply:
Changes Done